neqo-transport/src/recovery.rs

```diff
@@ -209,12 +227,152 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
+    congestion_window: u64, // = kInitialWindow
+    bytes_in_flight: u64,
+    congestion_recovery_start_time: Option<Instant>,
+    ssthresh: u64,
+}
+
+impl Default for CongestionControl {
+    fn default() -> Self {
+        CongestionControl {
+            congestion_window: INITIAL_WINDOW,
+            bytes_in_flight: 0,
+            congestion_recovery_start_time: None,
+            ssthresh: u64::max_value(),
+        }
+    }
+}
+
+impl Display for CongestionControl {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "CongCtrl")
+    }
+}
+
+impl CongestionControl {
+    #[cfg(test)]
```

Comment:
I don't see a problem with this being public more generally.  It's not actionable in the same way as `cwnd_avail`, but you could just document that limitation.

Reply:
What would be the benefit?